environment.yml

```diff
@@ -80,20 +76,20 @@ dependencies:
   - tqdm
   - yamllint
 
-  # Running NEST Server ----
-  - flask
-  - flask_cors
   - requests
-  - uwsgi
   - werkzeug
 
   # PIP dependencies -- do not delete
   - pip:
-    # For testsuite
-    - junitparser >= 2
-    # For documentation
-    - example
-    - Image
-    - sphinx_autobuild
-    - sphinx_gallery
-    - sphinx-tabs
+      # For testsuite
+      - junitparser >= 2
+      # For documentation
+      - example
+      - Image
+      - sphinx_autobuild
+      - sphinx_gallery
+      - sphinx-tabs
+      # For NEST Server
```

Comment:
Why did you move these dependencies to the pip section? Since we are providing a Conda environment file, I think we should only include packages not available on conda-forge in the `pip` section. 

Also, in the documentation above, the pip install command includes `RestrictedPython`. Should it be included here as well, as `restrictedpython` from conda-forge?

Reply:
I apologize. It was not intended to intent these lines. 
My code editor formats them automatically.

Is the new changes rather acceptable and usable?

Reply:
I cannot help with checking these changes in conda-forge.

Reply:
The `environment.yml` file seems to be fine now. I will test more thoroughly later.